plugins/wasi_nn/wasinn_openvino_genai.cpp

```diff
@@ -69,7 +70,10 @@ LLMPipelineBackend::SetContextInput(Context &CxtRef, uint32_t Index,
 
 Expect<WASINN::ErrNo> LLMPipelineBackend::Generate(Context &CxtRef) {
   try {
-    CxtRef.StringOutput = Model->generate(CxtRef.StringInput);
+    // TODO: let the user to set the generation config.
+    ov::genai::GenerationConfig config;
+    config.max_new_tokens = 100;
```

Comment:
We should have a warning message here to indicate that the current max_new_tokens is limited to only 100.